compiler/src/iree/compiler/Codegen/Dialect/GPU/TargetUtils/ConfigUtils.cpp

```diff
@@ -766,16 +767,15 @@ getMatmulOrIGEMMLoweringConfigAndWorkgroupSize(
       kPackFactor = std::get<2>(mmaKind.getMNKShape());
     }
     paddingTileSizes[innerKDim] *= kPackFactor;
+    attrs.emplace_back(StringAttr::get(context, "padding"),
+                       b.getI64ArrayAttr(paddingTileSizes));
```

Comment:
FYI, you can pass the name as stringref, only the value needs to be an attribute
```suggestion
    attrs.emplace_back("padding", b.getI64ArrayAttr(paddingTileSizes));
```

Reply:
also a few lines below